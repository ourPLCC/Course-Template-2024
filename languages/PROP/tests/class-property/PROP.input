define c =
class
field x
  property x = prop x:set x=$
end
define cc = new c
<cc>set x = 5 % ok - sets the field value to 5
<cc>x         % returns 5