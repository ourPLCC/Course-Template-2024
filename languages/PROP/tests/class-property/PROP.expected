c
cc
5
5
